src/utils/xml_parser.c

```diff
@@ -2590,3 +2603,501 @@ GF_XMLNode *gf_xml_dom_node_clone(GF_XMLNode *node)
 	}
 	return clone;
 }
+
```

Comment:
indeed we should move all these to src/utils/xml_bin_custom.c

Reply:
do you want me to also move gf_xml_parse_bit_sequence_bs() ?

Reply:
sure why not

Reply:
done